### DISCUSSION TITLE
a different CSS when using dark mode

### DISCUSSION THREAD

**Discussion by Laurent-Cosmic** [templaza/astroid-framework#380] [Q&A] [answered]
Hello
Is there a way to use a specific CSS when using dark mode?

I found a specific media but it's not working - does I miss something or Astroid is not returning this information?
@media (prefers-color-scheme: dark) {
    /* Dark theme styles go here */
    span.menuhomepage {
             color: red;
    }
}​

thanks in advance

**Comment by sonvnn** [comment, marked as answer]
Astroid Color Mode inherit from Bootstrap 5. You can check it here https://getbootstrap.com/docs/5.3/customize/color-modes/

If you would like style dark mode you can use Custom CSS https://docs.astroidframe.work/custom-code/custom-css

```
[data-bs-theme=dark] span.menuhomepage {
  color: red;
}
```

Best Regards,
Sonny

**Comment by Laurent-Cosmic** [comment]
thanks a lot
